actionpack/lib/action_dispatch/http/permissions_policy.rb

```diff
@@ -6,12 +6,7 @@ module ActionDispatch #:nodoc:
   class PermissionsPolicy
     class Middleware
       CONTENT_TYPE = "Content-Type"
-      # The Feature-Policy header has been renamed to Permissions-Policy.
-      # The Permissions-Policy requires a different implementation and isn't
-      # yet supported by all browsers. To avoid having to rename this
-      # middleware in the future we use the new name for the middleware but
-      # keep the old header name and implementation for now.
-      POLICY       = "Feature-Policy"
+      POLICY       = "Permissions-Policy"
```

Comment:
It seems we are no longer using this constant

Reply:
You're absolutely right. It's gone now.

Reply:
@rbgrouleff I was kind of asking because this constant is probably considered public API

Reply:
Oh, sorry. I didn't think about that. Could we keep `POLICY = "Permissions-Policy"` and add a `FEATURE_POLICY` constant as well perhaps with a comment explaining that both headers are added? Or is it better the other way around, since `Feature-Policy` is still better supported by browsers than `Permissions-Policy`?

Reply:
I think we shouldn't change the behavior of `POLICY` from FP unless we have a good reason. It sounds like "browser support" is a key concern we should respect.

Reply:
That makes sense, and adding a `PERMISSIONS_POLICY` now would mean it is just a matter of deprecating it, when it makes sense to no longer have the FP header.

Reply:
What is the point of having 2 constants?

Reply:
It would let us remove the `FeaturePolicyHeader.name` and `PermissionsPolicyHeader.name` methods, and developers relying on the `POLICY` would have a similar const to refer to for the PP.

Reply:
@rbgrouleff Sounds good to me!

Reply:
Great, `POLICY` is now back with.